src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/repository/BitbucketCloudTeam.java

```diff
@@ -39,6 +39,9 @@
     @JsonProperty("display_name")
     private String displayName;
 
+    @JsonProperty("uuid")
```

Comment:
There is getName() method in this class mapped to `@JsonProperty("username")`, this is going away. It should be mapped to "nickname" instead.

Reply:
So in this case, the BitbucketCloudTeam is retrieved with `getTeam()`. That method actually requesting from the teams api `.../2.0/teams/username`.  That path is staying as it will be only the team username.

Reply:
Let's leave it as it is. I think I was confused reading [teams api docs](https://developer.atlassian.com/bitbucket/api/2/reference/resource/teams/%7Busername%7D) suggested using nickname for team's name, but then calling their REST endpoint doesn't return nickname. I think what you have is good, no changes needed.

```
$ curl -v -u username:password  https://api.bitbucket.org/2.0/teams/vivektestteam | json_pp

{
   "links" : {
      "html" : {
         "href" : "https://bitbucket.org/vivektestteam/"
      },
      "avatar" : {
         "href" : "https://bitbucket.org/account/vivektestteam/avatar/"
      },
      "self" : {
         "href" : "https://api.bitbucket.org/2.0/teams/vivektestteam"
      }
   },
   "display_name" : "Vivek's Team",
   "type" : "team",
   "uuid" : "{47cd7cf2-ca31-4c90-bc0e-4c7ef67f9dfe}",
   "username" : "vivektestteam"
}
```